matter_server/Dockerfile

```diff
@@ -41,9 +39,6 @@ ARG MATTER_SERVER_VERSION
 
 # hadolint ignore=DL3013
 RUN \
-    pip3 install \
-       home-assistant-chip-clusters==${HOME_ASSISTANT_CHIP_VERSION} \
-       home-assistant-chip-core==${HOME_ASSISTANT_CHIP_VERSION} \
-    && pip3 install --no-cache-dir python-matter-server=="${MATTER_SERVER_VERSION}"
+    pip3 install --no-cache-dir python-matter-server[server]=="${MATTER_SERVER_VERSION}"
```

Comment:
is it safe to assume that these versions will stay on par ?

Reply:
Which versions? The SDK version in the add-on and the SDK version required by the server?

Reply:
ah yes, now I see. Instead of specifying which version of the chip sdk to include, you just choose which version of the python lib, much better!